win32/mxe/settings.mk

```diff
@@ -8,6 +8,6 @@ MXE_TARGETS := i686-w64-mingw32.shared
 #SOURCEFORGE_MIRROR := downloads.sourceforge.net
 # The three lines below makes `make` build these "local
 # packages" instead of all packages.
-LOCAL_PKG_LIST := gettext sdl sdl2 boost portaudio ffmpeg portmidi pango gdk-pixbuf librsvg libsigc++ libxml++ glew libcdio opencv libepoxy
+LOCAL_PKG_LIST := gettext sdl sdl2 boost portaudio ffmpeg portmidi pango gdk-pixbuf librsvg libsigc++ libxml++ glew libcdio opencv libepoxy jsoncpp openssl cpp-netlib
```

Comment:
@nieknooijens you also do need to add the epoxy_shared flag to cmakelist.txt else you won't be able to compile